db/migrate/20230620095911_add_search_to_keyword.rb

```diff
@@ -0,0 +1,16 @@
+class AddSearchToKeyword < ActiveRecord::Migration[7.0]
```

Comment:
```suggestion
class AddSearchToKeywords < ActiveRecord::Migration[7.0]
```

usually the `AddSearchCountsToKeyword ` is the table name. The file name also need to changed to `db/migrate/20230620095911_add_search_counts_to_keywords.rb` (with `s` after keyword)

Reply:
solved in 7e6b0a1